backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicatesHandler.java

```diff
@@ -110,7 +123,7 @@ private String getEntityName() {
                 .addCondition(String.format("%s.date_updated = :%s", alias, bindingName))
                 .addBind(q -> q.bind(bindingName, dateUpdated)));
       case "owner":
-        // case time created/updated:
```

Comment:
This was used because another PR had it, right?

Reply:
Yes